src/content/blog/ga_qas.md

```diff
@@ -0,0 +1,224 @@
+---
+title: An Introduction to GA-QAS
+author: Vu Tuan Hai
+day: 21
+month: 5
+year: 2024
+tags: 
+  - quantum architecture search
+  - python
+  - qiskit
+  - genetic-algorithm
+---
+
+In various quantum optimization problems, choosing the right ansatz is a critical point that will affect the result. Many template ansatzes have been proposed, such as Graph Ansatz and EffecientSU2, … but they are limited to the applications that you can apply. Then, we propose a search engine called **GA-QAS** (Genetic algorithm for quantum architecture search). This search engine can take your problem as input and return a good ansatz for you. In this post, we will guide you on how to use it efficiently.
+
+## Now, let’s start!
+
+# Step 1. Define problem (W state preparation)
+
+<img alt ='' src="/public/images/ga-qas/ga-qas1.png">
+
+_**Fig 1**. (a) State preparation scheme based on quantum compilation technique, where_ $U(\theta)$, $V^{\dagger}$ _are ansatz and state that need to prepare **[1][2]** (b) Some default ansatz, such as hyper-graph._
```

Comment:
You can combine this caption with the `img` tag above by using the following syntax like this:
```html
<figure>
    <img src='image.jpg' alt='description' />
    <figcaption>Caption goes here</figcaption>
</figure>
```

More details [here](https://stackoverflow.com/a/10128978).